src/Entities/Issue.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace ITBugTracking\Entities;
+
+use JsonSerializable;
+
+class Issue implements JsonSerializable
+{
+    public int $id;
+    public string $title;
+    public string|null $description;
+    public string $severity;
+    public string $date_created;
+    public string $reporter;
+    public int $department;
+    public int $completed;
```

Comment:
Would it be more useful to your application to either:
Store `$completed` as a boolean?
Use a getter to translate this internal `int` value to a boolean?

Reply:
done